### DISCUSSION TITLE
Buenas. estoy intentando descargar el producto para Windows exe. se instala pero no lo veo en pantalla

### DISCUSSION THREAD

**Discussion by Miguel196117** [ZL-Audio/ZLEqualizer#30] [General]
Buenas. estoy intentando descargar el producto para Windows exe. se instala pero no lo veo en pantalla

**Comment by zsliu98** [comment]
I am sorry I cannot read Spanish. Therefore, I will use the translation provided by GitHub. When you say "I don't see it on the screen", are you indicating that the plugin is not found by a DAW? I should address that the plugin is not a stand-alone app.

**Comment by Miguel196117** [comment]
Buenas de nuevo
No encuentro el software en la pantalla de inicio en mi ordenador.
Un sa

El lun, 25 mar 2024 a las 20:36, zsliu98 ***@***.***>)
escribió:

> I am sorry I cannot read Spanish. Therefore, I will use the translation
> provided by GitHub. When you say "I don't see it on the screen", are you
> indicating that the plugin is not found by a DAW? I should address that the
> plugin is not a stand-alone app.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ZL-Audio/ZLEqualizer/discussions/30#discussioncomment-8907543>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BHKKRK6BHTQ5DO4SH26FZMLY2CDE5AVCNFSM6AAAAABFHS5LYCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSMBXGU2DG>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Miguel Leon

**Reply by zsliu98** [reply]
It is installed into the VST3 path of your system. You have to use the **plugin** in a DAW (digital audio workstation). It does not function as a stand-alone app.
